cmd/controller/ctrl.go

```diff
@@ -84,6 +90,15 @@ func (c *Controller) Run(threadiness int, stopCh <-chan struct{}) error {
 		glog.Error("Unable to push disk objects to etcd : ", err)
 		return err
 	}
+	//This infinite loop will help to get event details send by udev-event module
+	//event details send using channel.
+	for true {
+		msg := <-UdevEventMessages
+		err := processDiskFromUdevEvent(c, msg)
+		if err != nil {
+			glog.Error(err)
```

Comment:
in error case, we should probably scans the disks here.

Reply:
done